goth/runner/__init__.py

```diff
@@ -174,19 +175,31 @@ def _get_current_test_name(self) -> str:
 
     async def _start_nodes(self):
         node_names: Dict[str, str] = {}
+        ports: Dict[str, dict] = {}
 
         # Start the probes' containers and obtain their IP addresses
         for probe in self.probes:
             await probe.start()
             assert probe.ip_address
-            node_names[probe.ip_address] = probe.name
+            ip = probe.ip_address
+            p = probe.container.ports
```

Comment:
Since we're not being constrained by line length limit here I'd avoid having a 1-letter variable name.